yt/data_objects/tests/test_dataset_access.py

```diff
@@ -1,15 +1,12 @@
 import numpy as np
-from nose.tools import assert_raises
+import pytest
```

Comment:
will have to add `test_dataset_access` to the nose ignores in `nose_ignores.txt` and `tests/tests.yaml`